src/Generator.cpp

```diff
@@ -47,15 +47,17 @@ const std::map<std::string, Halide::Type> &get_halide_type_enum_map() {
 }
 
 int generate_filter_main(int argc, char **argv, std::ostream &cerr) {
-    const char kUsage[] = "gengen [-g GENERATOR_NAME] [-f FUNCTION_NAME] [-o OUTPUT_DIR] [-r RUNTIME_NAME] [-e EMIT_OPTIONS] "
+    const char kUsage[] = "gengen [-g GENERATOR_NAME] [-f FUNCTION_NAME] [-o OUTPUT_DIR] [-r RUNTIME_NAME] [-e EMIT_OPTIONS] [-x EXTENSION_OPTIONS] "
                           "target=target-string [generator_arg=value [...]]\n\n"
                           "  -e  A comma separated list of optional files to emit. Accepted values are "
-                          "[assembly, bitcode, stmt, html]\n";
+                          "[assembly, bitcode, stmt, html, cpp]\n"
+                          "  -x  A comma separated list of extension=extension pairs to subsitute during file naming.\n";
```

Comment:
It means "I neglected to go back and clean this up before sending the PR". Oops. Will improve.